component/src/main/java/io/siddhi/extension/io/file/metrics/FileMoveMetrics.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.siddhi.extension.io.file.metrics;
+
+import org.wso2.carbon.metrics.core.Level;
+import org.wso2.carbon.si.metrics.core.internal.MetricsManagement;
+
+/**
+ * Class which is holds the metrics to monitor Move file operations.
+ */
+public class FileMoveMetrics extends Metrics {
+
+    private String source;
+    private String destination;
+    private long time;
+
+    public FileMoveMetrics(String siddhiAppName) {
+        super(siddhiAppName);
+    }
+
+    public void getMoveMetric(int status) {
+        MetricsManagement.getInstance().getMetricService()
+                .gauge(String.format("io.siddhi.SiddhiApps.%s.Siddhi.File.Operations.Move.%s.%s.%s",
```

Comment:
Cant change to have 1 metric class and pass "io.siddhi.SiddhiApps.%s.Siddhi.File.Operations.Move.%s.%s.%s" when creating and retrieving the metric?